src/main/java/science/icebreaker/config/ApplicationConfigurationProperties.java

```diff
@@ -0,0 +1,40 @@
+package science.icebreaker.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+
+@Configuration
+@ConfigurationProperties(prefix = "icebreaker")
+public class ApplicationConfigurationProperties {
```

Comment:
This class only contains configs regarding Authentication, thus the name should be changed and not kept so general.

Reply:
Well, since I don't think that we are doing to have so many different configuration properties, I kept the prefix general `icebreaker` instead of `icebreaker.security`. I would like to reuse this file for other configurations later.

Reply:
We will have more configurations quite soon. At the latest, if we want to server to send emails so that we have to provide the email login data.